server/src/main/java/org/cloudfoundry/identity/uaa/provider/IdentityProviderEndpoints.java

```diff
@@ -241,7 +344,7 @@ public ResponseEntity<String> testIdentityProvider(@RequestBody IdentityProvider
             status = BAD_REQUEST;
             exception = getExceptionString(x);
         } catch (Exception x) {
-            logger.error("Identity provider validation failed.", x);
+            logger.warn("Identity provider validation failed.", x);
```

Comment:
Why are we lowering this from an error to a warn?

Reply:
I used the `error` level for some of the log messages I added in the new code and reverted them - and accidentally also this one - back to `warn` later. I will change this particular call back to `error`, thanks for noticing :)

Reply:
fixed